.coafile

```diff
@@ -39,3 +39,8 @@ language = python3
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
 cs_keywords =
+
+
+[yml]
+bears = YAMLLintBear
+files = *.yml, .ci/*.yml, tests/**/*.yml
```

Comment:
There are two warnings:

```
[WARNING][12:59:53] No files matching '/home/ist/code/coala-vs-code/.ci/*.yml' were found. If this rule is not required, you can remove it from section [yml] in your .coafile to deactivate this warning.
[WARNING][12:59:53] No files matching '/home/ist/code/coala-vs-code/tests/**/*.yml' were found. If this rule is not required, you can remove it from section [yml] in your .coafile to deactivate this warning.
```

Could we remove `.ci/*.yml, tests/**/*.yml`?

Reply:
yes, right now there are no yml files there.But there may be in future? should i remove it?

Reply:
I think so, we could add them if we need them in the future :-)

Reply:
@gaocegege okay done